components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/SQLConstants.java

```diff
@@ -392,6 +392,9 @@ public class SQLConstants {
             "AND OH.DEPTH %s) ORDER BY UM_ORG_ATTRIBUTE.UM_ATTRIBUTE_KEY %s " +
             "OFFSET 0 ROWS FETCH NEXT :" + SQLPlaceholders.DB_SCHEMA_LIMIT + "; ROWS ONLY;";
 
+    public static final String GET_ORGANIZATION_NAME_BY_ID_MAP = "SELECT UM_ID, UM_ORG_NAME FROM UM_ORG " +
+            "WHERE UM_ID IN (";
```

Comment:
```suggestion
            "WHERE UM_ID IN (%s)";
```
Shall we have like this and add the placeholders later by using String.format() to improve the readability ?